### DISCUSSION TITLE
Part 2: Command line git discussion

### DISCUSSION THREAD

**Discussion by Ifycode** [Ifycode-experiments/git-github-training#88] [General]
Do you have any comments or questions concerning Part 2: Command line git learning videos and tasks? Type it in the comment, and someone will attend to you.

**Comment by samuko-things** [comment]
@Ifycode, there are no learning or task videos in the [documentation site](https://obiagba-mary.gitbook.io/git-and-github-training/part-2-command-line-git/command-line-git-setup)
